src/main/java/Duke.java

```diff
@@ -1,10 +1,61 @@
+import java.util.Scanner;
+import java.util.Arrays;
+
 public class Duke {
+    public static void printSeparator() {
+        String separator = "------------------------------------------";
```

Comment:
Good quality of code to make the line that are printed over and over into method so that it improves readability of codes.